File: src/test/java/euler/RomanNummeralsTest.java
package euler;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.is;
import static org.junit.Assert.assertThat;

public class RomanNummeralsTest {

  @Test
  public void testRom2Des() throws Exception {
    int des = RomanNummerals.romertall2tall("CMLXIX");
    assertThat( des, is(969));
    des = RomanNummerals.romertall2tall("MMXXVIII");
    assertThat( des, is(2028));
  }

  @Test
  public void konvertering(){
    String romertall = RomanNummerals.convert( 2399 );
    assertThat( romertall, is ("MMCCCXCIX"));
    romertall = RomanNummerals.convert( 49 );
    assertThat( romertall, is ("XLIX"));
  }
}

File: src/test/scala/euler/RomanNummeralsTest.scala
package euler

import org.scalatest.FunSuite

class RomanNummeralsTest extends FunSuite {

  test("Rom2Des") {
    val des = RomanNummerals.romertall2tall("CMLXIX")
    assert(des == 969)
    val des2 = RomanNummerals.romertall2tall("MMXXVIII")
    assert(des2 == 2028)
  }

  test("konvertering") {
    val romertall = RomanNummerals.convert(2399)
    assert(romertall == "MMCCCXCIX")
    val romertall49 = RomanNummerals.convert(49)
    assert(romertall49 == "XLIX")
  }
}
